frontend/src/app/components/header/header.component.ts

```diff
@@ -27,6 +29,10 @@ export class HeaderComponent implements OnInit, OnDestroy {
     this.subscription = this.authService.isUserLoggedIn.subscribe((value: boolean) => {
       this.isActiveButtonsAuth = !value;
     });
+
+    this.subscription = this.authService.isAdminLoggedIn.subscribe((value: boolean) => {
```

Comment:
1. You rewrite which you se 29 line. You can't do it
2. You should create 1 dunction which will check who is logged in. In your current implementation if application will have 20 roles it means you will create here 20 functions for checking roles. It is bad